### ISSUE TITLE
Create new logo - check with Ingrid

### ISSUE THREAD

**Issue by nanaynaomi** [nanaynaomi/multnomah-falls-research#3]
1/10/2022

Create some various logo designs to replace the other logo and/or change the coloring of the old logo. Confirm with Ingrid before starting this. Once logo designs are created, show them to Ingrid to hear her thoughts on them and how we could change them to work with the site.

**Comment by nanaynaomi** [comment]
1/11/2022

Created new logo. Waiting to hear Ingrid's thoughts on it.

**Comment by nanaynaomi** [comment]
1/20/2022

She likes the logo, but we both agree that it is a bit busy so we might just use this one somewhere else and have something simpler with just "MFR" and a little super simple logo (something that is kind of like a mix between a waterfall and fiber optic sensors) on the nav bar.

**Comment by nanaynaomi** [comment]
1/20/2022

Closing this because I did remake the main logo but now I need to do a different logo for the nav bar - making a new issue for that.
